backend/benefit/applications/services/pdf_templates/associations.html

```diff
@@ -0,0 +1,40 @@
+<html>
+<meta http-equiv="Content-type" content="text/html; charset=utf-8" />
```

Comment:
Is it possible to factor the templates so that the common code (`<head>` and such) would be only defined once?